airbyte/destinations/_util.py

```
@@ -17,23 +17,40 @@
 if TYPE_CHECKING:
     from collections.abc import Callable
 
     from airbyte.caches.base import CacheBase
+    from airbyte.destinations import Destination
 
 
 SNOWFLAKE_PASSWORD_SECRET_NAME = "SNOWFLAKE_PASSWORD"
 
 
+def cache_class_to_destination_class(
+    cache_class: type[CacheBase],
```

Ruff (0.8.0)

Unused function argument: `cache_class`

(ARG001)